.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -4,6 +4,13 @@ about: Submit a bug report to help us improve Habitat
 
 ---
 
+## Habitat-Sim version
+vx.x.x or master?
+
+Habitat is under active development, and we advise users to restrict themselves to stable releases. Are you using the latest [release version](https://github.com/facebookresearch/habitat-sim/releases) of Habitat-Sim? The bug you are about to report may already be fixed there.
```

Comment:
Leaving this here too:  I am not sure where "there" is in already fixed there.  Do you mean stable or latest version?